src/l/struct.658

```diff
@@ -715,7 +715,7 @@ PG$		MRA PAGING
 		MRA PPN
 20$ 		MRA PS
 IFN ITS,[
-		MRA [ITS,AI,ML,MC,MD,MX,DB,KA,KL,HX]
+		MRA [ITS,AI,ML,MC,MD,MX,DB,KA,KL,HX,TT]
```

Comment:
@bictorv, did you consider this list of machine names for replacement with something else?

Reply:
That code just creates the atoms (MRA=Make Random Atom) which were needed by the old code (in macro expansion), but the new code creates the atom for the local host using SIXATM, so no need to create them beforehand.

Reply:
Actually the list could probably/better be removed in STRUCT (possibly retaining just ITS).